feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/input/ui/audiorecorder/AudioRecorder.kt

```diff
@@ -44,6 +50,9 @@ fun AudioRecorderScreen(
 ) {
   val audioRecorderViewState by audioRecorderViewModel.viewState.collectAsState()
 
+  val openPermissionDialog = remember { mutableStateOf(false) }
```

Comment:
I think it's a bit more natural to write these as 
```suggestion
  var openPermissionDialog by remember { mutableStateOf(false) }
```
So that you can just edit the value itself instead of having to call `.value` on it.

Having it as a var immediately shows us that we have control over its value and can edit it locally. If it's a val, you have to look inside the remember lambda, see that it's a `mutableState` and then you know you can edit it. Because you may well have a `remember { false }` which is just an immutable `false` value which is not editable.

So I think these are some compelling reasons to follow that style 😊

Reply:
Fixing!